src/Nethermind/Nethermind.Optimism/OptimismPlugin.cs

```diff
@@ -61,6 +63,13 @@ public class OptimismPlugin(ChainSpec chainSpec) : IConsensusPlugin, ISynchroniz
     private OptimismCL? _cl;
     public bool Enabled => chainSpec.SealEngineType == SealEngineType;
 
+    public IEnumerable<StepInfo> GetSteps()
+    {
+        yield return new StepInfo(typeof(InitializeBlockchainOptimism));
+        yield return new StepInfo(typeof(InitializeBlockProducerOptimism));
+        yield return new StepInfo(typeof(RegisterOptimismRpcModules));
+    }
```

Comment:
Minor syntatic suggar add implicit casting from `Type` to `StepInfo` and change this to 
```suggestion
        yield return typeof(InitializeBlockchainOptimism);
        yield return typeof(InitializeBlockProducerOptimism);
        yield return typeof(RegisterOptimismRpcModules);
    }
```

Reply:
Done